src/main/resources/templates/macros/report/tags.vm

```diff
@@ -2,7 +2,7 @@
 
 <div class="tags indention">
   #foreach($tag in $tags)
-     <a href="$tag.getFileName()">$tag.getName()</a>
+     <a class="btn btn-default btn-xs" href="$tag.getFileName()">$tag.getName()</a>
```

Comment:
can you use style so it is visible from the first look that this element is clickable?